CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [1.38.109](https://github.com/brave/brave-browser/releases/tag/v1.38.109)
+
+ - Added Gemini two-way Brave Rewards wallet. ([#17408](https://github.com/brave/brave-browser/issues/17408))
+ - Added the ability to redirect an AMP page to its canonical non-AMP version. ([#21643](https://github.com/brave/brave-browser/issues/21643))
+ - Added the ability to copy Brave Wallet transaction hash from "Transactions" on the "Portfolio" page. ([#19886](https://github.com/brave/brave-browser/issues/19886))
+ - Added the ability to set custom nonce value for Brave Wallet transactions. ([#20887](https://github.com/brave/brave-browser/issues/20887))
+ - Added the ability to customize the nonce for unapproved Brave Wallet transactions via the transactions confirmation screen. ([#21459](https://github.com/brave/brave-browser/issues/21459))
+ - Added "Clear wallet transaction and nonce information" setting for Brave Wallet under brave://settings/wallet. ([#21457](https://github.com/brave/brave-browser/issues/21457))
```

Comment:
that one is on Android but we don't have `brave://settings/wallet` it's just inside Settings menu

Reply:
Fixed 👍